webClient/src/app/editor/code-editor/monaco/monaco.component.ts

```diff
@@ -40,10 +44,35 @@ export class MonacoComponent implements OnInit, OnChanges {
   }
 
   ngOnInit() {
+    (<any>window).monaco.baseUrl = '/ZLUX/plugins/org.zowe.editor/web/';
```

Comment:
You created a 'pubPath' for this

Reply:
I really don't know what this does. I tried a lot of things